third-party/realdds/include/realdds/dds-device.h

```diff
@@ -51,10 +51,14 @@ class dds_device
     size_t number_of_streams() const;
 
     size_t foreach_stream( std::function< void( std::shared_ptr< dds_stream > stream ) > fn ) const;
+    size_t foreach_option( std::function< void( std::shared_ptr< dds_option > option ) > fn ) const;
 
     void open( const dds_stream_profiles & profiles );
     void close( const dds_streams & streams );
 
+    void set_option_value( std::shared_ptr< dds_option > option, float value );
```

Comment:
I think these will need to be `json` values eventually -- next PR, I guess?

Reply:
But also:
Have we though about being able to query the option value from the option itself?